base/common/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -939,10 +971,23 @@ public static X509Key createX509Key(PublicKey publicKey) throws InvalidKeyExcept
                     params.getQ(),
                     params.getG());
 
+        } else if (publicKey instanceof ECPublicKey) {
+
+            ECPublicKey ecPublicKey = (ECPublicKey) publicKey;
+            try {
+                DerValue derValue = new DerValue(ecPublicKey.getEncoded());
+                return X509Key.parse(derValue);
+            } catch (IOException e) {
+                throw new InvalidKeyException(e);
+            }
+
```

Comment:
Just curious.  Is there a reason why the     "    } else if (publicKey instanceof ECPublicKey) {"  was moved down to after DSAPublicKey?